multisig-transactions-monitor/src/utils.js

```diff
@@ -0,0 +1,470 @@
+    );
+  }
+
+  return finding;
+}
+
+/// CREATE FINDINGS FOR COMPTROLLER INTERACTIONS ///
+
+/*
+        "NewPauseGuardian",
+        "ActionPaused",
+        "NewBorrowCap",
+        "NewBorrowCapGuardian",
+        "NewPauseGuardian"
+*/
+
+function createNewPauseGuardianFinding(
+  log,
+  protocolName,
+  protocolAbbreviation,
+  developerAbbreviation,
+) {
+  const { oldPauseGuardian, newPauseGuardian } = log.args;
+
+  const finding = Finding.fromObject({
+    name: `${protocolName} New Pause Guardian`,
+    description: `Pause Guardian changed from ${oldPauseGuardian} to ${newPauseGuardian}`,
+    alertId: `${developerAbbreviation}-${protocolAbbreviation}-NEW-PAUSE-GUARDIAN-ALERT`,
+    protocol: config.protocolName,
+    type: FindingType.Info,
+    severity: FindingSeverity.Info,
+    metadata: {
+      oldPauseGuardian,
+      newPauseGuardian,
+      multisigAddress,
+    },
+  });
+  return finding;
+}
+
+function createActionPausedFinding(
+  log,
+  protocolName,
+  protocolAbbreviation,
+  developerAbbreviation,
+) {
+  const { action } = log.args;
+
+  const finding = Finding.fromObject({
+    name: `${protocolName} Action Paused`,
+    description: `Action ${action} was Paused by multisig ${multisigAddress}`,
+    alertId: `${developerAbbreviation}-${protocolAbbreviation}-ACTION-PAUSED-ALERT`,
+    protocol: config.protocolName,
+    type: FindingType.Info,
+    severity: FindingSeverity.Info,
+    metadata: {
+      action,
+      multisigAddress,
+    },
+  });
+  return finding;
+}
+
+function createNewBorrowCapFinding(
+  log,
+  protocolName,
+  protocolAbbreviation,
+  developerAbbreviation,
+) {
+  const { cToken, newBorrowCap } = log.args;
+
+  const finding = Finding.fromObject({
+    name: `${protocolName} New Borrow Cap`,
+    description: `New Borrow Cap for cToken ${cToken} with a borrow cap of ${newBorrowCap} was set by multisig ${multisigAddress}`,
+    alertId: `${developerAbbreviation}-${protocolAbbreviation}-NEW-BORROW-CAP-ALERT`,
+    protocol: config.protocolName,
+    type: FindingType.Info,
+    severity: FindingSeverity.Info,
+    metadata: {
+      cToken,
+      newBorrowCap,
+      multisigAddress,
+    },
+  });
+  return finding;
+}
+
+function createNewBorrowCapGaurdianFinding(
+  log,
+  protocolName,
+  protocolAbbreviation,
+  developerAbbreviation,
+) {
+  const { oldBorrowCapGuardian, newBorrowCapGuardian } = log.args;
+
+  const finding = Finding.fromObject({
+    name: `${protocolName} New Borrow Cap Guardian`,
+    description: `Borrow Guardian changed from ${oldBorrowCapGuardian} to ${newBorrowCapGuardian}`,
+    alertId: `${developerAbbreviation}-${protocolAbbreviation}-NEW-BORROW-CAP-GUARDIAN-ALERT`,
+    protocol: config.protocolName,
+    type: FindingType.Info,
+    severity: FindingSeverity.Info,
+    metadata: {
+      oldBorrowCapGuardian,
+      newBorrowCapGuardian,
+      multisigAddress,
+    },
+  });
+  return finding;
+}
+
+function createComptrollerFinding(
```

Comment:
Same comment as for the previous function.  You could use a `switch` statement in place of the `if` statements, but it's fine either way.